server/services/auth_service.go

```diff
@@ -100,3 +102,29 @@ func GetUserFromSession(c *fiber.Ctx) (*models.User, error) {
 
 	return user, nil
 }
+func InsertUserToSession(c *fiber.Ctx, user *models.User) int {
+	sess, err := GetStore(c)
+	statusCodeError := fiber.StatusInternalServerError
```

Comment:
Don't user `statusCodeError` the service shouldn't know anything about status codes, only controller